sdk/src/Handlers/EntityFramework/AWSXRayRecorder.Handlers.EntityFramework.csproj

```diff
@@ -23,6 +23,10 @@
     <NoWarn>1701;1702;1591;</NoWarn>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <NoWarn>1701;1702;1591;</NoWarn>
```

Comment:
Any reason why you're not including all of the following warning codes for both debug and release?
1701;1702;1705;1591;1587;1572;1573